src/main/resources/view/HelpWindow.fxml

```diff
@@ -64,31 +64,34 @@
                <font>
                   <Font name="System Bold" size="17.0" />
                </font></Label>
-            <TableView fx:id="tableView" prefHeight="315.0" prefWidth="589.0" GridPane.rowIndex="3">
+            <TableView fx:id="tableView" minHeight="-Infinity" minWidth="-Infinity" prefHeight="315.0" prefWidth="589.0" GridPane.rowIndex="3">
                 <columns>
-                    <TableColumn prefWidth="88.0" text="Action">
+                    <TableColumn prefWidth="88.0" resizable="false" text="Action">
```

Comment:
I think this should be called command right?

Reply:
good catch, fixed in the following commit